internal/widget/scroller.go

```diff
@@ -82,16 +82,17 @@ func (b *scrollBar) Cursor() desktop.Cursor {
 }
 
 func (b *scrollBar) DragEnd() {
-	b.area.isDragged = false
+	b.area.isDragging = false
 
 	if fyne.CurrentDevice().IsMobile() {
 		b.area.MouseOut()
 	}
+	b.area.Refresh()
```

Comment:
I think this is causing a double refresh on mobile. You might want to add a return inside the if-statement.

Reply:
Done